pages/common/cargo-locate-project.md

```diff
@@ -0,0 +1,21 @@
+# cargo locate-project
+
+> Print the full path to the `Cargo.toml` manifest of the current project.
+> If the project is part of a workspace, the manifest of the project is shown, rather than the workspace output.
```

Comment:
This sounds like it shows the output of a workspace, and there's no such thing.
```suggestion
> If the project is part of a workspace, the manifest of the project is shown, rather than that of the workspace.
```